topic_folders/communications/guides/submit-job-posting.md

```diff
@@ -0,0 +1,41 @@
+## Sharing Job Postings
```

Comment:
suggest changing this from _Sharing Job Postings_ to _Sharing Opportunities_ to include sharing of fellowship, scholarship and grant opportunities